src/js/components/DataChart/stories/Axes.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, DataChart, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { data } from './data';
```

Comment:
It could be more straight forward if the data could be included in the file.

Reply:
agreed, done